addons/sale_management/tests/test_sale_onboarding.py

```diff
@@ -0,0 +1,14 @@
+import odoo.tests
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+class TestOnboarding(odoo.tests.HttpCase):
+    def test_01_get_sample_sales_order_from_scratch(self):
+        # Make sure there are no QO nor products
+        self.env['sale.order'].search([
+            ('company_id', '=', self.env.company.id),
+            ('partner_id', '=', self.env.user.partner_id.id),
+            ('state', '=', 'draft')
+        ]).state = 'cancel'
+        self.env['product.product'].search([]).active = False
+        self.env.company._get_sample_sales_order()
```

Comment:
why didn't you specify this test in the `sale` module ?
Considering the method we want to test is there, the test should be too :thinking: